KukaApp/screens/TestimoniesScreen/TestimonyCard.js

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+import { View } from 'react-native';
+import {
+  Button,
+  Divider,
+  StyleService,
+  Text,
+  useStyleSheet,
+} from '@ui-kitten/components';
+import { Bar } from 'react-native-progress';
+import TestimonyStatusIcon from './TestimonyStatusIcon';
+import BookIcon from '../../assets/images/book.svg';
+
+const Header = ({ title, billCode, position }) => {
+  return (
+    <View style={{ padding: 20, display: 'flex', flexDirection: 'row' }}>
+      <View style={{ width: '90%' }}>
+        <Text category="h6">{title}</Text>
+        <View style={{ display: 'flex', flexDirection: 'row', marginTop: 2 }}>
+          <BookIcon style={{ marginRight: 5 }} fill="#8F9BB3" width={10} />
+          <Text appearance="hint" style={{ fontSize: 12, lineHeight: 20 }}>
+            {billCode}
+          </Text>
+        </View>
+      </View>
+      <View style={{ width: '10%' }}>
+        <TestimonyStatusIcon userPosition={position} />
+      </View>
+    </View>
+  );
+};
+
+const Footer = () => (
+  <View
+    style={{
+      flex: 1,
+      flexDirection: 'row',
+      justifyContent: 'space-around',
+      padding: 15,
+    }}
+  >
+    <Button appearance="outline" status="basic" size="small">
+      HEARING DETAILS
+    </Button>
+    <Button appearance="outline" size="small">
+      TESTIMONY
+    </Button>
+  </View>
+);
+
+/**
+ *
+ * @param agendaItem
+ * @param testimony
+ * @param uploadPercentComplete, floating point 0% = 0.0, 50% = 0.5, 100% = 1.0
+ * @returns {JSX.Element}
+ * @constructor
+ */
+export const TestimonyCard = ({
+  agendaItem,
+  testimony,
+  uploadPercentComplete,
+}) => {
+  const styles = useStyleSheet(themedStyles);
+  const position = uploadPercentComplete < 1.0 ? 'sync' : testimony.position;
+
+  return (
+    <View style={styles.card}>
+      <View>
+        {Header({
```

Comment:
A pet peeve of mine. Choose between `<Header ...agendaItem, position={position} />` or make a function like `renderHeader(..agendaItem, position)`.

Reply:
Same goes for Footer down there.